index.html

```diff
@@ -53,6 +53,60 @@
   <link rel="stylesheet" href="./Css-Files/testimonial.css" />
   <script src="chatboxx.js" defer></script>
   <link rel="stylesheet" href="Css-Files/caduceus.css">
+  <!-- <script src="https://cdn.emailjs.com/dist/email.min.js"></script> -->
+  <script type="text/javascript"
+        src="https://cdn.jsdelivr.net/npm/@emailjs/browser@4/dist/email.min.js">
+  </script>
+    Copy code

```

Comment:
Can remove this line?

Reply:
Yeah but unable to commit the changes

Reply:
I think this line must there..
Is it giving any warnings..